specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -832,44 +942,36 @@ model DimensionValue {
   value?: string;
 }
 
-@resource("values")
-@parentResource(Dimension)
+@doc("Metrics dimension values.")
 model DimensionValueList {
```

Comment:
should the dimensionValueList also have the name of the dimension?

Reply:
This is response for get /test-runs/{testRunId}/metric-dimensions/{name}/values which requires dimension name as path parameter. So, it is good to have dimension name in response but the response list is already for a specific dimension.